catalog/core/catalog-core-impl/pubsub/src/main/java/ddf/catalog/pubsub/EventProcessorImpl.java

```diff
@@ -120,10 +124,10 @@ public EventProcessorImpl(
    */
   public static void processEntry(Metacard metacard, String operation, EventAdmin eventAdmin) {
     String methodName = "processEntry";
-    LOGGER.debug("ENTERING: " + methodName);
+    LOGGER.debug(ENTERING, methodName);
```

Comment:
looks like this needs `{}`

Reply:
@bdeining I made the ENTERING constant to have the "{}" in it.

Reply:
saw that; my mistake 😬